### ISSUE TITLE
Declare other OS support

### ISSUE THREAD

**Issue by alexcom** [Koteyk0o/LYWSD02MMC-Desktop-App#2]
It would be useful to know if this software can be built and used on non-Windows systems.

**Comment by whoisnotthere** [comment]
Yep, you can download/clone project and build for Linux. I tested with Ubuntu 18.04, works great

**Comment by alexcom** [comment]
FYI: I've tried on macOS. The build succeeds but the app crashes when I press "Connect".

**Comment by alexcom** [comment]
Addition regarding macOS: if I give the app Bluetooth access permission beforehand via System Preferences->Security&Privacy->Privacy the app starts to scan successfully, finds some devices but their MAC addresses are all zeros. When trying to connect to device the process either times out or crashes(on successful attempt?).

**Comment by whoisnotthere** [comment]
Ok, i will find macOS computer, and will try to fix this. I'll post an update a little later.

**Comment by gpunktschmitz** [comment]
it would also be great if you could add the Linux build to the releases, or add instructions on how one could build it for on Linux themselves.
